FAQ/FAQ-LightningNetwork.md

```diff
@@ -204,6 +205,12 @@ There are numerous ways to find your node information, but the easiest way to di
 
 ![BTCPay Checkout](/img/LightningNodepPageInfo.png)
 
+### Where can I find recovery seed backup for my Lightning Network wallet in BTCPay Server?
+
+BTCPay uses noseedbackup, so you can’t backup your LN wallet or get your recovery seed.This is because in Lightning Network there’s still no solution for backing up funds in channels, just in your on-chain wallet.
```

Comment:
```suggestion
BTCPay uses noseedbackup, so you can’t backup your LN wallet or get your recovery seed. This is because in Lightning Network there’s still no solution for backing up funds in channels, just in your on-chain wallet.
```